web/react/components/user_settings/manage_incoming_hooks.jsx

```diff
@@ -183,7 +183,7 @@ export default class ManageIncomingHooks extends React.Component {
             <div key='addIncomingHook'>
                 <FormattedHTMLMessage
                     id='user.settings.hooks_in.description'
-                    defaultMessage='Create webhook URLs for use in external integrations. Please see<a href="http://mattermost.org/webhooks" target="_blank">http://mattermost.org/webhooks</a> to learn more.'
+                    defaultMessage='Create webhook URLs for use in external integrations. Please see <a href="http://mattermost.org/webhooks" target="_blank">http://mattermost.org/webhooks</a> to learn more.'
```

Comment:
That's done automatically by using the tool, so no worries with that. actually when using the tool you dont need to include the strings in en.json for the frontend side